services/paybuttonsService.ts

```diff
@@ -3,7 +3,7 @@ import { PayButton } from 'types'
 import * as chainService from 'services/chainsService'
 
 export async function createPaybutton (userId: string, prefixedAddressList: string[]): Promise<PayButton>  {
-    const result = await models.sequelize.transaction(async (t) => {
+    const result = await models.sequelize.transaction(async (t: any) => {
```

Comment:
Lets try to keep linting of existing files to their own PR's, as linting only PR's need very little review.

Reply:
done, please git pull @chedieck